tests/compat.sh

```diff
@@ -790,14 +818,21 @@ wait_client_done() {
     echo "EXIT: $EXIT" >> $CLI_OUT
 }
 
+# print_test_title <CLIENT> <SERVER> <STANDARD_CIPHER_SUITE>
+print_test_title() {
+    for i in $3; do
+        TITLE="$1->$2 $MODE,$VERIF $i"
+        DOTS72="........................................................................"
+        printf "%s %.*s " "$TITLE" "$((71 - ${#TITLE}))" "$DOTS72"
```

Comment:
This will print dots when we are listing the test cases.

Reply:
Right. I intended to print extra dots in option `--list-test-case` in order to use common code to display output. Now I realize we can combine line 827 to ignore the extra dots.